goc/share/oasis-batch-worker

```diff
@@ -17,11 +17,9 @@ do
 done
 
 ## determine the vo requesting the update
-vo=$1
-proxy=$2
-startip=$3
+vo="$1"
```

Comment:
Just a minor note, but for variable assignments like this (`vo=$1`), the quotes are not actually necessary.